app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -116,4 +121,36 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
     companion object {
         fun newInstance() = SettingsFragment()
     }
+
+    override fun showDownloadDialog(data : String?) {
+        ui {
+            val context = this
+            val builder = AlertDialog.Builder(activity)
+            val view = layoutInflater.inflate(R.layout.dialog_download, null)
+            if (data.equals("UserDataDownload_RequestExisted_Text"))
```

Comment:
Not a big deal to declare this statement like this.

Reply:
Improved the code.